herddb-core/src/main/java/herddb/sql/DDLSQLPlanner.java

```diff
@@ -821,6 +832,54 @@ private Statement buildExecuteStatement(String defaultTableSpace, Execute execut
             default:
                 throw new StatementExecutionException("Unsupported command " + execute.getName());
         }
+
+    }
+
+    public Statement QueryConsistencyCheckStatement(String defaultTablespace, String query, List<Object> parameters) {
```

Comment:
in Java method names start with a lowercase character /QueryConsistencyCheckStatement/queryConsistencyCheckStatement/